ql/src/java/org/apache/hadoop/hive/ql/txn/compactor/handler/TaskHandlerFactory.java

```diff
@@ -43,7 +44,14 @@ private TaskHandlerFactory() {
 
   public List<TaskHandler> getHandlers(HiveConf conf, TxnStore txnHandler, MetadataCache metadataCache,
                                                   boolean metricsEnabled, FSRemover fsRemover) {
-    return Arrays.asList(new CompactionCleaner(conf, txnHandler, metadataCache,
+    boolean useAbortHandler = MetastoreConf.getBoolVar(conf, MetastoreConf.ConfVars.COMPACTOR_CLEAN_ABORTS_USING_CLEANER);
+    List<TaskHandler> taskHandlers = new ArrayList<>();
+    if (useAbortHandler) {
```

Comment:
no need for that check, from now on use Cleaner to handle aborts

Reply:
This is part of the cleanup task when the config will be removed. Until then this is supposed to be here.